SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -646,6 +646,15 @@ void GenerateActorSchema(FComponentIdGenerator& IdGenerator, UClass* Class, TSha
 
 	ActorClassPathToSchema.Add(Class->GetPathName(), ActorSchemaData);
 
+	// Cache the NCD for this Actor
```

Comment:
What is this actor? The default actor?

Reply:
The default actor for this specific class type. CDO (Class Default Objects) are used to instantiate additional actors of a specific type at runtime efficiently.